examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>Calibrations are accessible via the <code>native_gate_calibration</code> property of the device object. The calibrations are downloaded and cached when the property is first referenced. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>